lambda_function/outputs.tf

```diff
@@ -4,4 +4,8 @@ output "arn" {
 
 output "role" {
   value = aws_iam_role.lambda
-}
\ No newline at end of file
+}
+
+output "function_name" {
+  value = aws_lambda_function.function_name
```

Comment:
I think this references a non-existent resource from the module and maybe should be `aws_lambda_function.lambda.function_name`

Reply:
that makes sense. Made change.